crypto/keys/ed25519/ed25519.go

```diff
@@ -17,6 +17,8 @@ import (
 var _ crypto.PrivKey = PrivKey{}
 
 const (
+	PrivKeyName = "cosmos-sdk/PrivKeyEd25519"
```

Comment:
BTW this is breaking the encoding. If you want to keep backwards compatibility with amino encoding you have to keep the previous key names.